README.md

```diff
@@ -1,88 +1,87 @@
+### Contribution Guidelines
 
-[Imperative CLI Framework](https://github.com/zowe/imperative/wiki) documentation is a key source of information to learn about the features of Imperative CLI Framework (the code framework that you use to build plug-ins for Zowe CLI). Refer to these documents during development.
+The Zowe CLI [contribution guidelines](CONTRIBUTING.md) contain standards and conventions for developing Zowe CLI Plug-ins.
 
-### Jenkinsfile Guidelines
+The guidelines contain critical information about working with the code, running/writing/maintaining automated tests, developing consistent syntax in your plug-in, and ensuring that your plug-in integrates with Zowe CLI properly.
 
-Reference the [Jenkinsfile Guidelines](CICD-TEMPLATE.md) for information about setting up and maintaining automated testing/deployment for your plug-in with Jenkins automation server.
+### CICD Guidelines
 
-## Prerequisites
+Reference our [included GitHub Actions workflows](.github/workflows/) for guidance about setting up and maintaining automated testing and deployment for your plug-in with GitHub Actions.
 
-Before you work with the Zowe CLI sample plug-in, [install Zowe CLI globally.](https://docs.zowe.org/active-development/user-guide/cli-installcli.html)
+## Set up your Development Environment
 
-## Create a Local Development Space
+### Prerequisites
 
-To create your development space, clone and build the Zowe CLI sample plug-in from source.
+[Install Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli) and its dependencies globally.
 
-Create a local development folder named `zowe-tutorial`. You will clone and build all projects in this folder.
+### Initial Setup
 
-Clone the repositories into your development folder to match the following structure:
+To create your development space, you will clone and build the zowe-cli-sample-plugin from source.
 
+Clone the repository into your development folder to match the following structure:
 ```
 zowe-tutorial
 └── zowe-cli-sample-plugin
 ```
 
-### Clone zowe-cli-sample-plugin and Build From Source
-
-See [setup](docs/tutorials/Setup.md).
-
-### Run the Automated Tests
-
-**Note:** If you don't have access to a z/OSMF instance at your site, run `npm run server:start` to launch a mock server at http://localhost:3000.
-
-1. `cd __tests__/__resources__/properties`
-2. Copy `example_properties.yaml` to `custom_properties.yaml`.
-3. Edit the properties within `custom_properties.yaml` to contain valid system information for your site.
-4. `cd` to your `zowe-cli-sample-plugin` folder
-5. `npm run test`
-
-### Install the zowe-cli-sample-plugin to Zowe CLI
+1. `cd` to your `zowe-tutorial` folder
+2. `git clone https://github.com/zowe/zowe-cli-sample-plugin`
+3. `cd zowe-cli-sample-plugin`
+4. `npm install`
+5. `npm run build`
 
-This process assumes that you already installed Zowe CLI on your PC in the previous steps.
+## Next steps: Using your Sample Plug-in
```

Comment:
Do we capitalize every word in titles?
I remember having to fix this in other plug-ins 😋 
cc @anaxceron

Reply:
i dont think words like `your` need capitalization but idk

Reply:
@zFernand0 @ATorrise -- I don't think so b/c they're specific commands. You could say "Enter `blah` command to change your directory to `abc`. Or: Issued `blah` to clone the `plugin` repository.

Reply:
fixing all titles